README.md

```diff
@@ -108,6 +108,19 @@ $ cargo doc
 
 The API reference can be found in `target/doc/amethyst/index.html`.
 
+
+## Questions / Help
+
+Please check out the [FAQ][faq] before asking.
+
+If you have an easy question, just ask on [Gitter][gt] and we'll help you and add it to the FAQ.
+
+For more complex questions, [r/rust_gamedev][rg] might be the right place.
```

Comment:
Sounds good to me! We could also add the #rust-gamedev irc channel (double check the channel name please).

> Thought it would flood gitter if somebody has a bigger project.

I would prefer to root all amethyst questions to the gitter as it's also feedback about the library and the current design!